src/json.json

```
@@ -0,0 +1,54 @@
+{
+    "id": "6283598291e6b519379c26b4",
+    "propertyId": "2033",
+    "jurisdiction": null,
```

```suggestion
    "jurisdiction": null,
```
look for a new way of suggestion